sources/europe/fi/tampere-aerial-2022.geojson

```diff
@@ -0,0 +1,226 @@
+{
+    "type": "Feature",
+    "properties": {
+        "attribution": {
+            "required": true,
+            "text": "© City of Tampere",
+            "url": "https://www.tampere.fi/en"
+        },
+        "available_projections": [
+            "EPSG:2391",
+            "EPSG:3386",
+            "EPSG:2391",
+            "EPSG:2392",
+            "EPSG:2393",
+            "EPSG:2394",
+            "EPSG:2397",
+            "EPSG:4123",
+            "EPSG:3878",
+            "EPSG:3067",
+            "EPSG:3126",
+            "EPSG:3127",
+            "EPSG:3128",
+            "EPSG:3129",
+            "EPSG:3130",
+            "EPSG:3131",
+            "EPSG:3132",
+            "EPSG:3133",
+            "EPSG:3134",
+            "EPSG:3135",
+            "EPSG:3136",
+            "EPSG:3137",
+            "EPSG:3138",
+            "EPSG:3034",
+            "EPSG:3035",
+            "EPSG:3046",
+            "EPSG:3047",
+            "EPSG:3048",
+            "EPSG:4258",
+            "EPSG:4937",
+            "EPSG:4936",
+            "EPSG:5717",
+            "EPSG:4326",
+            "EPSG:3785",
+            "EPSG:3857",
+            "CRS:84"
+        ],
+        "best": true,
+        "country_code": "FI",
+        "description": "Orthorectified aerial imagery of Tampere (excl. Aitolahti and Teisko) and Pirkkala (excl. airport). Taken in May 2022, resolution of this layer is 12.5 cm.",
+        "icon": "https://www.tampere.fi/themes/custom/tampere/images/favicon-files/tre-favicon-96-x-96.png",
+        "id": "tampere-aerial-2022",
+        "license_url": "https://github.com/osmlab/editor-layer-index/files/14606823/Tampere_wms_waiver.pdf",
```

Comment:
Is that from a file uploaded to a GitHub Issue or PR description or comment? To ensure we don't loose it, could it either be uploaded to the OSM Wiki or placed directly in `sources/europe/fi`?